sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_mldesigner/__init__.py

```diff
@@ -0,0 +1,121 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+"""
+This file stores functions and objects that will be used in mldesigner package.
+DO NOT change the module names in "all" list. If the interface has changed in source code, wrap it here and keep
+original function/module names the same as before, otherwise mldesigner will be broken by this change.
+"""
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
+
+from azure.ai.ml import(
+    Input,
+    Output,
+    MLClient,
+    load_component,
+    MpiDistribution,
+    PyTorchDistribution,
+    TensorFlowDistribution,
+)
+from azure.ai.ml.entities import(
+    Component,
+    CommandComponent,
+    PipelineComponent,
+    Environment,
+    ResourceConfiguration,
+    ValidationResult,
+    Command,
+    Pipeline,
+    Parallel,
+)
+from azure.identity import AzureCliCredential
+from azure.ai.ml.exceptions import ErrorTarget, UserErrorException
+from azure.ai.ml.entities._component.component_factory import component_factory
+from azure.ai.ml.entities._job.pipeline._load_component import _generate_component_function
+from azure.ai.ml._internal.entities._additional_includes import _AdditionalIncludes
+from azure.ai.ml._utils.utils import load_yaml
+from azure.ai.ml.entities._validation import SchemaValidatableMixin
+from azure.ai.ml._schema import PathAwareSchema
+from azure.ai.ml.dsl._utils import _sanitize_python_variable_name
+from azure.ai.ml._internal.entities import (
+    InternalComponent,
+    Ae365exepool,
+    Command as InternalCommand,
+    Parallel as InternalParallel,
+    DataTransfer,
+    Distributed,
+    HDInsight,
+    Hemera,
+    Scope,
+    Starlite,
+)
+
+component_factory_load_from_dict = component_factory.load_from_dict
+
+assert hasattr(Component, "_customized_validate")
```

Comment:
Not sure if we put these assertions here or in test file, these are the assertions we hope no one changes but test file tends to let people think they can change accordingly?

Reply:
Put it in test files as discussed in meeting